### PR TITLE
KAA-1585: Account profile can be updated with an invalid email

### PR DESCRIPTION
**Pull request by Acarus**

### CODE REVIEW

**Comment by iamtodor** [comment]
@Acarus please, follow field naming conventions: 
public(private) static final fields (constants) are ALL_CAPS_WITH_UNDERSCORES.

**Comment by Acarus** [comment]
@iamtodor All that code written in that style (not by me) I woun't change such amount of code because of one field. (https://github.com/Acarus/kaa/blob/84f99193366bf81e50d2ddbcf1e5198e9342f6b9/server/node/src/main/java/org/kaaproject/kaa/server/admin/client/util/Utils.java#L50)

**Comment by Acarus** [comment]
@kaa-leeroy retest this please

**Comment by garniy** [comment]
@kaa-leeroy retest this please

**Comment by sashadidukh** [comment]
@kaa-leeroy retest this please

**Comment by Oldnickus** [comment]
@kaa-leeroy skip

**Comment by Oldnickus** [comment]
@kaa-leeroy skip ci

**Comment by Oldnickus** [comment]
@kaa-leeroy retest this please

File: server/node/src/main/java/org/kaaproject/kaa/server/admin/client/util/Utils.java [hunk 1]
```diff
@@ -174,14 +176,7 @@ public static String millisecondsToDateTimeString(long millis) {
...
   public static boolean validateEmail(String mail) {
-    boolean result = false;
-    if (mail != null && mail.length() != 0) {
-      if (mail.indexOf('@') != INCORRECT_IDX
-          && mail.indexOf('.') != INCORRECT_IDX) {
-        result = true;
-      }
-    }
-    return result;
+    return !(mail == null || mail.length() == 0) && emailPattern.exec(mail) != null;
```
Comment (rasendubi) [inline server/node/src/main/java/org/kaaproject/kaa/server/admin/client/util/Utils.java:179]: Would you like to apply De Morgan's rule?
`!(mail == null || mail.length() == 0)` -> `mail != null && mail.length != 0`

**Comment by Oldnickus** [comment]
skip ci

**Comment by garniy** [comment]
Jenkins and Travis fails are not related. 
Pull request already merged into the release-0.10 branch

**Comment by garniy** [comment]
@rasendubi, can we merge this PR to master?

**Comment by rasendubi** [comment]
@sergio-dev I'm not blocking it. (Though Jenkins hasn't passed yet and I see no approve reviews.)

**Review by garniy — APPROVED** [review]

**Comment by garniy** [comment]
@rasendubi, Jenkins cannot start this build. 
After discussion with @Oldnickus I decide to merge this PR

**Pull request merged**